src/Item/Item.service.ts

```diff
@@ -56,7 +56,15 @@ export class ItemService {
     eth_address: string
   ): Promise<FullItem> {
     let dbCollection: CollectionAttributes | undefined = undefined
-    const dbItem = await Item.findOne<ItemAttributes>(item.id)
+    const decodedItemURN =
+      !item.id && item.urn ? decodeThirdPartyItemURN(item.urn) : null
+    const dbItem = decodedItemURN
+      ? await Item.findByURNSuffix(
+          decodedItemURN.third_party_id,
+          decodedItemURN.item_urn_suffix
+        )
```

Comment:
If we don't fail when we're updating an item by URN and the item doesn't exist in the DB, we won't be able to tell the user that the editing process failed as it's always going to be creating a new item.
How does `upsertThirdPartyItem` behave when the item id and the item don't exist?